lib/openshift/flakes/conditions.rb

```diff
@@ -0,0 +1,40 @@
+module BushSlicer
+
+  # this class should help with parsing report conditions
+  class Conditions
```

Comment:
this class appears to represent a single condition so a name `Condition` (without `s`) would make more sense. If it is specific for a particular object type, then it might be called `ReportCondition`. But it looks like generic so if I am not mistaken `Condition` should be fine.